ajo/personalization-client-side/README.md

```diff
@@ -0,0 +1,112 @@
+# Client-side Personalization
+
+## Overview
+
+This sample demonstrates using Adobe Experience Platform to get personalization content from Adobe Journey Optimizer. The web page changes based on the personalization content returned.
+
+This sample uses the [Adobe Experience Platform Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) to get personalization content and to render it entirely client-side.
+
+Here is what the page looks like before and after personalization content is rendered.
+
+| without personalization                                        | with personalization                                                 |
+|----------------------------------------------------------------|----------------------------------------------------------------------|
+| <img src="../../.assets/plain.png" alt="drawing" width="800"/> | <img src="../../.assets/with-offers.png" alt="drawing" width="800"/> |
+
+Please review the [summary of personalization content](../../ajo/TargetActivities.md) for this sample.
+
+Keep in mind that personalization content can come from any Adobe source such as [Journey Optimizer](https://experienceleague.adobe.com/docs/journey-optimizer/using/ajo-home.html?lang=en) or [Target](https://experienceleague.adobe.com/docs/target.html). Target is the source of personalization content used in this sample, but the implementation described here will work for all personalization solutions Adobe offers.
```

Comment:
Adobe Journey Optimizer is the source of personalization content used ...

Reply:
Done